File: android/ValleCOM/app/src/main/java/com/valleapp/vallecom/utilidades/Instruccion.java
package com.valleapp.valleCOM.utilidades;

import android.content.ContentValues;
import android.os.Handler;

public class Instruccion {

    public ContentValues getParams() {
        return params;
    }

    public String getUrl() {
        return url;
    }

    public Handler getHandler() {
        return handler;
    }

    public String getOp() {
        return op;
    }

    String url;
    Handler handler;
    ContentValues params;
    String op = "";

    public Instruccion(ContentValues params, String url, Handler handler, String op){
        this.params = params; this.url = url; this.handler = handler; this.op = op;
    }
    public Instruccion(ContentValues params, String url){
        this.params = params; this.url = url; this.handler = null;
    }
}

File: android/ValleTPV/ValleTPVlib/src/main/java/com/valleapp/valletpvlib/tools/Instruccion.kt
package com.valleapp.valletpvlib.tools

import android.content.ContentValues
import android.os.Handler
import java.util.UUID

class Instruccion(
    val params: ContentValues,
    val url: String,
    val handler: Handler? = null, // Handler es opcional
    val op: String = ""
) {
    init {
        params.put("uid_device", UUID.randomUUID().toString())
    }

}
